app/services/invoices/progressive_billing_service.rb

```diff
@@ -99,7 +99,7 @@ def invoiced_amount_cents
       @invoiced_amount_cents ||= subscription.invoices
         .finalized
         .where(invoice_type: %w[subscription progressive_billing])
-        .sum(:fees_amount_cents)
+        .sum { |invoice| invoice.fees.charge.sum(:amount_cents) }
```

Comment:
`invoice.fees.charge` Is it correct to only look at `charge` fees? (e.g. we create dedicated `progressive_billing` fees for the progressive invoices, I thought)

Reply:
Good catch, I will update it!